docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -65,18 +76,18 @@ The following example shows how to create a gallery-like flexbox layout using th
     toolBar.JustifyContent = FlexContainer.Justification.JustifySpaceBetween;
     ```
 
-5.  Use the toolbar's `Flex` property to make the toolbar and the main content share the height of the main container, so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder:
+6.  Use the toolbar's `Flex` property to make the toolbar and the main content share the height of the main container, so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder:
```

Comment:
I couldn't get this clearly but tried to comprehend what is being conveyed, kindly check if this is as per your intention, or else can you slightly elaborate? 

```suggestion
6.  Use the `Flex` property of the toolbar to make the toolbar. The main content shares the height of the main container so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder space: